src/main/java/com/jfrog/ide/idea/ui/webview/WebviewManager.java

```diff
@@ -43,6 +49,16 @@ public void onLoadError(CefBrowser browser, CefFrame frame, ErrorCode errorCode,
         }, jbCefBrowser.getCefBrowser());
     }
 
+    private void streamConsoleMessagesToLog() {
+        jbCefBrowser.getJBCefClient().addDisplayHandler(new CefDisplayHandlerAdapter() {
+            @Override
+            public boolean onConsoleMessage(CefBrowser browser, CefSettings.LogSeverity level, String message, String source, int line) {
```

Comment:
Should we also consider the `CefSettings.LogSeverity` in our logger?
For instance, logging `LOGSEVERITY_VERBOSE` only in debug?